.github/workflows/db-update.yml

```diff
@@ -72,7 +72,7 @@ jobs:
       uses: liquibase-github-actions/update@v4.23.0
       with:
         classpath: 'liquibase'
-        changeLogFile: 'clear_tables.xml'
+        changeLogFile: 'population_prep_tables.xml'
```

Comment:
Do we still need clear_tables.xml?

Reply:
I thought it might be useful for clearing dev for example or any local dev env